src/models/events/event-models.js

```diff
@@ -46,6 +47,20 @@ function remove(id) {
   return db("Events").where({ id }).del();
 }
 
+async function findUninvitedUsersForEvent(id) {
+  const allUsers = await db("Users");
+
+  const invitedUsers = await db("Users")
+    .select("*")
+    .join("Events_Status", "Events_Status.user_id", "Users.id")
+    .where("Events_Status.event_id", id);
+
+  const seen = {};
+  invitedUsers.forEach(user => seen[user.id] = user);
+
+  return allUsers.filter(user => !(user.id in seen));
+}
+
```

Comment:
is this logic to find uninvited users not possible on front end without this new model? seems you could use all users as base, and filter out if they on on invited list there. Perhaps it is more convoluted since you must go through events to get to users for each pass?

Reply:
I could definitely try switching the logic over to the front end instead. I was just getting duplicate users on the front end when editing and wanted to get a filtered list.

Reply:
well, this seems to work with relative ease, so that works for me.  Theoretically, I think with other models, we have the pieces to get the same list without it, but this may prove useful in other situations as well. Thanks for writing it all out.